### ISSUE TITLE
LED animation not matching the jingle notes (to slow)

### ISSUE THREAD

**Issue by piebrain** [piebrain/G3-FW#2]

**Comment by piebrain** [comment]
@pymutrux is this still a problem?

It will probably become a problem whenever we tinker with the startup code...

**Comment by pymutrux** [comment]
Yes, it would be nice to align the LED turning on with the sound. But that is low priority.

**Comment by pymutrux** [comment]
Here are the Start, Stop and Duration times for each note of the ON and OFF chimes (v 3.1):

ON
	Start		Durat	Stop
0	0		35		35
1	35		315		350
2	350		310		660
3	660		305		965
4	965		1535	2500

OFF
	Start		Durat	Stop
0	0		30		30
1	30		335		365
2	365		315		680
3	680		305		985
4	985		1515	2500

All values in ms
